apps/prairielearn/src/pages/partials/studentGroupControls.ejs

```diff
@@ -16,39 +16,8 @@
 <%- include('../shared/groupWorkInitial.ejs', {groupConfig: groupConfig}); %>
 <% } %>
 <% } else {%>
-  <%- include('../shared/groupWorkInfoContainer.ejs', {groupConfig: groupConfig, groupMembers: groupMembers, groupName: groupName}); %>
-  <% if (groupConfig.has_roles) { %>
-    <% if (!rolesInfo.rolesAreBalanced) { %>
-      <div class="alert alert-danger my-2" role="alert">
-        A user has too many roles. Every student should be assigned to exactly <strong>one</strong> role with group size <strong><%= groupSize %></strong> 
-      </div>
-    <% } %>
-    <% if (typeof rolesInfo.validationErrors !== 'undefined' && rolesInfo.validationErrors.length > 0) { %>
-      <% rolesInfo.validationErrors.forEach(function ({ role_id, role_name, count, minimum, maximum }) { %>
-        <div class="alert alert-danger my-2" role="alert">
-          <% if (count < minimum) { %>
-            <%= minimum - count %> more <%= minimum - count === 1 ? 'person needs' : 'people need' %> to be assigned <%= role_name %>.
-          <% } else if (count > maximum) { %>
-            <%= count - maximum %> less <%= count - maximum === 1 ? 'person needs' : 'people need' %> to be assigned <%= role_name %>.
-          <% } %>
-          <% if (maximum === minimum) { %>
-            (Found <%= count %>, expected exactly <%= minimum %>).
-          <% } else { %>
-            (Found <%= count %>, expected between <%= minimum %> and <%= maximum %>).
-          <% } %>
-        </div>
-      <% }) %>
-    <% } %>
-    <% if (typeof rolesInfo.usersWithoutRoles !== 'undefined' && rolesInfo.usersWithoutRoles.length > 0) { %>
-      <div class="alert alert-danger my-2" role="alert">
-        At least one user does not have a role. All users must have a role.
-      </div>
-    <% } %>
-    <% if (canViewRoleTable) { %>
-    <%- include('../shared/groupRoleSelectTable.ejs', {groupMembers: groupMembers, rolesInfo: rolesInfo}); %>
-    <% } %>
-  <% } %>
-  <% if (locals.assessment.type === 'Exam' && locals.assessment.require_honor_code) { %>
+  <%- include('./groupWorkInfoContainer.ejs'); %>
+  <% if (start && locals.assessment.type === 'Exam' && locals.assessment.require_honor_code) { %>
```

Comment:
I can't wait til this is all strongly typed. It took me way too long to figure out what `start` is and where it comes from. (Also IMO that's a horrible name for this variable, though I know it was that way before you started.)